packages/devtools_app/lib/src/framework/notifications_view.dart

```diff
@@ -114,7 +114,7 @@ class _NotificationsState extends State<_Notifications> with AutoDisposeMixin {
     bool didDismiss = false;
     // Make a copy so we do not remove a notification from [_notifications]
     // while iterating over it.
-    final notifications = List<_Notification>.from(_notifications);
+    final notifications = List<_Notification>.of(_notifications);
```

Comment:
was this caught by a lint? what is the lint preventing?

Reply:
This is a lint suggested at https://github.com/dart-lang/linter/issues/2555
that Dart Code Metrics implemented.